tests/deployment_utils.py

```diff
@@ -205,20 +206,20 @@ def _vcs_args(self, vcs_type):
     def run(self, config, force_installed=False, vcs_type="none",
             additional_parameters="", environment=None, expected_to_fail=False):
 
-        config_file = os.path.join(self.working_dir, "temp_config.py")
-        with open(config_file, 'wb+') as f:
-            f.write(config)
-            f.close()
-
-        cmd = "coverage run --branch --append --source='{}' '{}/universum.py'" \
-            .format(self.working_dir, self.working_dir)
+        cmd = "coverage run --branch --append --source='{0}' '{0}/universum.py'" \
+            .format(self.working_dir)
 
         # We cannot collect coverage from installed module, so we run it only if specifically told so
         if force_installed:
             cmd = "universum"
 
-        cmd += " -lcp '{}'".format(config_file) \
-               + self._basic_args() + self._vcs_args(vcs_type) + additional_parameters
+        if self.command == 'nonci':
+            cmd += ' nonci'
+        else:
+            cmd += self._vcs_args(vcs_type)
```

Comment:
This code seems to allow using subcommands other then 'nonci'; but passing any other command will lead to wrong argument set and failure.
I'd rather implement full support of subcommands or pass 'nonci' as flag instead of multi-value string.